CMakeLists.txt

```diff
@@ -54,9 +54,7 @@ if(NOT DEFINED ENABLE_LTO)
   set(ENABLE_LTO OFF)
 endif()
 
-if(NOT DEFINED DISABLE_ES2015)
-  set(FEATURE_PROFILE "es2015-subset")
-endif()
```

Comment:
The above lines should not be removed, I think adding the default FEATURE_PROFILE for ShadowNode would be better :)